tests/components/stookwijzer/test_config_flow.py

```diff
@@ -24,20 +31,29 @@ async def test_full_user_flow(hass: HomeAssistant) -> None:
     ) as mock_setup_entry:
         result2 = await hass.config_entries.flow.async_configure(
             result["flow_id"],
-            user_input={
-                CONF_LOCATION: {
-                    CONF_LATITUDE: 1.0,
-                    CONF_LONGITUDE: 1.1,
-                }
-            },
+            user_input=CONF_INPUT,
         )
 
-    assert result2.get("type") is FlowResultType.CREATE_ENTRY
-    assert result2.get("data") == {
-        "location": {
-            "latitude": 1.0,
-            "longitude": 1.1,
-        },
-    }
+    assert result2.get("type") == FlowResultType.CREATE_ENTRY
+    assert result2.get("data") == CONF_DATA
 
     assert len(mock_setup_entry.mock_calls) == 1
+
+
+async def test_flow_while_unavailable(
+    hass: HomeAssistant, aioclient_mock: AiohttpClientMocker
+) -> None:
+    """Test the full user configuration flow while unavailable."""
+    mock_transform_failure(aioclient_mock)
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": SOURCE_USER, "version": 1}
+    )
+
+    with patch(
+        "homeassistant.components.stookwijzer.async_setup_entry", return_value=True
+    ):
+        await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            user_input=CONF_INPUT,
+        )
```

Comment:
What is this testing? This is just touching code

Reply:
🤷  I have no clue myself either

Reply:
handled.